src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -278,12 +276,6 @@ public void error(ExecutorDriver driver, String message) {
     log.error(this.getClass().getName() + ".error: " + message);
   }
 
```

Comment:
Yes, what you added definitely made it much better. Essentially, we want to stop launching any tasks and add a timeout as well to make sure the tasks have time to get shutdown. See this thread, http://mail-archives.us.apache.org/mod_mbox/mesos-dev/201411.mbox/%3CCACzFoUsKMqEESrNTK-Hp_SKpGUu11qg+og4_bE6+Ld9e5+ROsw@mail.gmail.com%3E. I think we can safely keep the TODO assigned to me. You might make a note about adding a timeout period. @adam-mesos may also have some input here.